<launch>

	<!-- <arg name="robot_ip"          default="192.168.100.77"/>
	<arg name="network_interface" default="eno1" />
	<node name="dynparam_load" type="dynparam" pkg="dynamic_reconfigure" args="load /tf_dynreconf_node $(find mild_hri_ros)/config/nuitrack_pepper_tf.yaml"/>

	<include file="$(find moveint_ros)/launch/pepper_bringup.launch">
		<arg name="robot_ip" value="$(arg robot_ip)"/>
		<arg name="network_interface" value="$(arg network_interface)"/>
	</include> -->

	<node pkg="nodelet" type="nodelet" args="manager" name="standalone_nodelet" output="screen"/>
	<node pkg="nodelet" type="nodelet" name="depth_image_proc" args="load depth_image_proc/point_cloud_xyzrgb standalone_nodelet --no-bond">
			<remap from="rgb/camera_info" to="camera_info"/>
			<remap from="rgb/image_rect_color" to="image_color"/>
			<remap from="depth_registered/image_rect" to="image_depth"/>
			<remap from="depth_registered/points" to="pointcloud2/points"/>
    </node>

	<!-- <node name="tf_dynreconf_node" type="node" pkg="tf_dynreconf" />
	<node name="check_selfcollision" pkg="check_selfcollision" type="service_node" respawn="true" output="screen"/> -->

	<!-- <node name="rviz" type="rviz" pkg="rviz" args="-d $(find mild_hri_ros)/config/hri.rviz"/> -->

	<!-- <node name="image_view" type="image_view" pkg="image_view">
		<remap from="image" to="image_display"/>
	</node> -->

	<!-- External webcam for the experimenter to view the interaction -->
	<!-- <node name="webcam_view" type="webcam_view.py" pkg="mild_hri_ros"/> -->


</launch>
